src/components/challengeWidget/challengeWidget.css

```diff
@@ -0,0 +1,32 @@
+.challengeWidget {
+  width: 700px;
+  height: 250px;
+  border: 15px solid burlywood;
+  background-color: white;
+}
+
+h1 {
```

Comment:
We'll have to "namespace" all CSS rules in here under `. challengeWidget`.
So for example instead of `h2`, we'll need `. challengeWidget h2`.
This is to ensure that other `<h2>` elements in the app are not affected.